frontend/src/scenes/groups/GroupsIntroduction.tsx

```diff
@@ -21,15 +21,16 @@ export function GroupsIntroduction({ access }: Props): JSX.Element {
             data-attr="group-analytics-upgrade"
             className="groups-introduction__action-button"
         >
-            Upgrade to get Group Analytics
+            Upgrade to get Group analytics
         </LemonButton>
     )
 
     const learnMoreButton = (
         <LemonButton
-            type={access === GroupsAccessStatus.HasAccess ? 'primary' : undefined}
-            to="https://posthog.com/docs/user-guides/group-analytics?utm_medium=in-product&utm_campaign=group-analytics-learn-more"
+            type={access === GroupsAccessStatus.HasAccess ? 'primary' : 'highlighted'}
+            to="https://posthog.com/docs/user-guides/group-analytics?utm_medium=in-product&utm_campaign=group-analytics-page"
             target="_blank"
+            rel="noopener"
```

Comment:
Why do we need this?

From https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener

> Note: Setting target="_blank" on <a> elements now implicitly provides the same rel behavior as setting rel="noopener" which does not set window.opener. See browser compatibility for support status.

Reply:
Cool! Did not know this